LICENSE.txt

```diff
@@ -108,6 +108,10 @@ Copyright (C) 2014 Justin Aquadro (MIT):
   textures/drawers_wood_front_2.png
   textures/drawers_wood_front_4.png
   textures/drawers_wood.png
+  
+Copyright (C) 2018 isaiah658 (CC0):
+  textures/drawer_controller_side.png
+  textures/drawer_controller_top_bottom.png
```

Comment:
Why don't you use the textures from the original mod? (It's ok of course to use yours :smiley:)